lib/proxy/plugin_initializer.rb

```diff
@@ -227,22 +274,12 @@ def merge_settings(provider_settings, main_plugin_settings)
     provider_settings.merge(main_plugin_settings)
   end
 
-  def used_settings(settings)
+  def log_used_settings(settings)
     default_settings = plugin.plugin_default_settings
     sorted_keys = settings.keys.map(&:to_s).sort.map(&:to_sym) # ruby 1.8.7 doesn't support sorting of symbols
     sorted_keys.map {|k| "'%s': %s%s" % [k, settings[k], (default_settings.include?(k) && default_settings[k] == settings[k]) ? " (default)" : ""] }.join(", ")
```

Comment:
This isn't getting logged anywhere, I think you intended to move the logger.debug into this method?